thoth/storages/graph/postgres.py

```diff
@@ -2514,6 +2514,23 @@ def get_python_package_version_dependents_all(
 
             return result
 
+    def python_package_version_depends_on_platform_exists(self, platform: str) -> bool:
+        """Check if the given platform has some records in the database."""
+        with self._session_scope() as session:
+            return session.query(exists().where(DependsOn.platform == platform)).scalar()
+
+    def get_python_package_version_platform_all(self) -> List[str]:
```

Comment:
Hope these two methods conform to the naming convention.

Reply:
Yes they do :1st_place_medal: